tests/hpxml_translator_test.rb

```diff
@@ -298,6 +298,14 @@ def _test_simulation(args, this_dir, rundir, expect_error, expect_error_msgs)
   end
 
   def _verify_simulation_outputs(rundir, hpxml_path, results)
+    # Check that eplusout.err has no lines that include "Blank Schedule Type Limits Name input"
+    File.readlines(File.join(rundir, "eplusout.err")).each do |err_line|
+      next if err_line.include? 'Schedule:Constant="ALWAYS ON CONTINUOUS", Blank Schedule Type Limits Name input' # FIXME
+      next if err_line.include? 'Schedule:Constant="ALWAYS OFF DISCRETE", Blank Schedule Type Limits Name input' # FIXME
```

Comment:
What's the deal with these two? They are provided by OS and don't have schedule type limits provided?

Reply:
These two schedules exist in the idf file (without schedule type limits assigned) but do not exist in the osm. Perhaps OS forward translates these schedules regardless of their existence in the osm?

Reply:
Or perhaps we have a reporting measure that's injecting it?

Reply:
Probably forward translator, yeah. You can go ahead and remove the FIXME comments.

Reply:
Although if it is the forward translator, I still wonder what it's forward translating these schedules based on? Because I only see this for a small subset of the test xml files...